lib/ret_web/channels/hub_channel.ex

```diff
@@ -18,11 +27,11 @@ defmodule RetWeb.HubChannel do
     socket |> assign(:profile, profile) |> assign(:context, context) |> perform_join(hub_sid)
   end
 
-  defp perform_join(socket, hub_sid, push_subscription_endpoint \\ nil) do
+  defp perform_join(socket, hub_sid, push_subscription_endpoint \\ nil, token \\ nil) do
```

Comment:
we probably want to refer to tokens consistently o/w it is gonna get confusing:

- `access_token` - I think this is what we're using for the file upload bits
- (proposed) `auth_token` - Token for authentication
- (proposed) `perms_token` - Token for authorization (not relevant yet)